CHANGELOG.md

```diff
@@ -6,30 +6,35 @@ release lines.
 
 Select a Node.js version below to view the changelog history:
 
-* [Node.js v7](doc/changelogs/CHANGELOG_V7.md)
-* [Node.js v6](doc/changelogs/CHANGELOG_V6.md)
-* [Node.js v5](doc/changelogs/CHANGELOG_V5.md)
-* [Node.js v4](doc/changelogs/CHANGELOG_V4.md)
-* [Node.js v0.12](doc/changelogs/CHANGELOG_V012.md)
-* [Node.js v0.10](doc/changelogs/CHANGELOG_V010.md)
+* [Node.js 8](doc/changelogs/CHANGELOG_V8.md)
+* [Node.js 7](doc/changelogs/CHANGELOG_V7.md)
+* [Node.js 6](doc/changelogs/CHANGELOG_V6.md)
+* [Node.js 5](doc/changelogs/CHANGELOG_V5.md)
+* [Node.js 4](doc/changelogs/CHANGELOG_V4.md)
+* [Node.js 0.12](doc/changelogs/CHANGELOG_V012.md)
+* [Node.js 0.10](doc/changelogs/CHANGELOG_V010.md)
 * [io.js](doc/changelogs/CHANGELOG_IOJS.md) and [Archive](doc/changelogs/CHANGELOG_ARCHIVE.md)
 
 Please use the following table to find the changelog for a specific Node.js
 release.
 
 <table>
 <tr>
-  <th><a href="doc/changelogs/CHANGELOG_V7.md">v7</a><sup>Current</sup></th>
-  <th title="LTS Until 2019-04"><a href="doc/changelogs/CHANGELOG_V6.md">v6</a><sup>LTS</sup></th>
-  <th title="Unsupported Since 2016-07-01"><a href="doc/changelogs/CHANGELOG_V5.md">v5</a><sup>EOL</sup></th>
-  <th title="LTS Until 2018-04"><a href="doc/changelogs/CHANGELOG_V4.md">v4</a><sup>LTS</sup></th>
-  <th title="Unsupported Since 2017-01-01"><a href="doc/changelogs/CHANGELOG_V012.md">v0.12</a><sup>EOL</sup></th>
-  <th title="Unsupported Since 2016-11-01" colspan="3"><a href="doc/changelogs/CHANGELOG_V010.md">v0.10</a><sup>EOL</sup></th>
+  <th><a href="doc/changelogs/CHANGELOG_V8.md">8</a><sup>Current</sup></th>
+  <th><a href="doc/changelogs/CHANGELOG_V7.md">7</a><sup>-</sup></th>
+  <th title="LTS Until 2019-04"><a href="doc/changelogs/CHANGELOG_V6.md">6</a><sup>LTS</sup></th>
+  <th title="Unsupported Since 2016-07-01"><a href="doc/changelogs/CHANGELOG_V5.md">5</a><sup>EOL</sup></th>
+  <th title="LTS Until 2018-04"><a href="doc/changelogs/CHANGELOG_V4.md">4</a><sup>LTS</sup></th>
+  <th title="Unsupported Since 2017-01-01"><a href="doc/changelogs/CHANGELOG_V012.md">0.12</a><sup>EOL</sup></th>
+  <th title="Unsupported Since 2016-11-01" colspan="3"><a href="doc/changelogs/CHANGELOG_V010.md">0.10</a><sup>EOL</sup></th>
 </tr>
 <tr>
     <td valign="top">
+<b><a href="doc/changelogs/CHANGELOG_V8.md#8.0.0">8.0.0</a></b><br/>
+    </td>
+    <td valign="top">
 <b><a href="doc/changelogs/CHANGELOG_V7.md#7.10.0">7.10.0</a></b><br/>
```

Comment:
This line and the one below it need to be changed to remove the bold tags I think?